theme/js/404.js

```diff
@@ -16,14 +16,14 @@ $(function() {
         var redirectRule = redirects[i],
             fromURLRedirect = redirectRule.from.toLowerCase();
 
-        // Note this is a case insensitive (exact) string match trailing slashes may break the comparison
+        // Note this is a case-insensitive match for the given pattern.
         if (_currentPath.indexOf(fromURLRedirect) >= 0 && typeof redirectRule.to === 'string') {
-          var targetURL = _redirectMap.defaultBaseToURL + redirectRule.to;
```

Comment:
Removing `defaultBaseToURL` made it possible to use relative links that don't depend on the docs site being deployed to any particular domain. It is still possible to set absolute URLs in `redirects.json` if we need to redirect somewhere else.